keras/src/callbacks/orbax_checkpoint_test.py

```diff
@@ -0,0 +1,1536 @@
+import os
+import shutil
+import tempfile
+
+import numpy as np
+import pytest
+
+from keras.src import backend
+from keras.src import layers
+from keras.src import models
+from keras.src import testing
+
+try:
+    # Import advanced Orbax functionality through the Keras bridge
+    from keras.src.callbacks.orbax_checkpoint import CheckpointManager
+    from keras.src.callbacks.orbax_checkpoint import OrbaxCheckpoint
+    from keras.src.callbacks.orbax_checkpoint import PyTreeCheckpointer
+    from keras.src.callbacks.orbax_checkpoint import SaveArgs
+    from keras.src.callbacks.orbax_checkpoint import StandardRestore
+    from keras.src.callbacks.orbax_checkpoint import TypeHandler
+    from keras.src.callbacks.orbax_checkpoint import metadata
+    from keras.src.callbacks.orbax_checkpoint import register_type_handler
+except ImportError:
+    OrbaxCheckpoint = None
+    CheckpointManager = None
+    SaveArgs = None
+    StandardRestore = None
+    TypeHandler = None
+    register_type_handler = None
+    PyTreeCheckpointer = None
+    metadata = None
```

Comment:
For unit tests, it's ok to fail on import. This unit test will require orbax to be installed.

Reply:
ok